tests/lib/rules/no-ember-data-global-imports.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-ember-data-global-imports');
+const RuleTester = require('eslint').RuleTester;
+const parserOptions = { ecmaVersion: 6, sourceType: 'module' };
+
+const message = 'Imports from @ember-data packages should be prefered over imports from ember-data';
```

Comment:
Instead of duplicating this error message here, we should import it from the rule file. Take a look at the `no-test-and-then` rule to see an example of how this importing can be done.

Reply:
This is done now